client/src/common/Navigation/Button.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+import PropTypes from "prop-types";
+/**
+ * Button is a component that is responsible for being the universally styles ALL
+ * navigation button. This component is to be used as a single source of truth for button
+ * implementation.
+ */
+const Button = (props) => {
+  const {
+    buttonText,
+    onClick,
+    key,
+    disabled = false,
+    isPrimary = true,
+  } = props;
+  let style = "btn btn-primary text-black btn-xl text-uppercase";
```

Comment:
I know we were talking with Saige about switching away from all uppercase text, do we want to remove the `text-uppercase` styling here?

Reply:
Leaving this particular issue for the Auxiliary team.